app/(dashboard)/mailboxes/[mailbox_slug]/settings/tools/metadataEndpointSetting.tsx

```
@@ -15,6 +14,7 @@ import { Label } from "@/components/ui/label";
 import { Tooltip, TooltipContent, TooltipProvider, TooltipTrigger } from "@/components/ui/tooltip";
 import { api } from "@/trpc/react";
 import SectionWrapper from "../sectionWrapper";
+import { SecretInput } from "@/components/secretInput";
```

ESLint

[error] Delete `";⏎import·{·SecretInput·}·from·"@/components/secretInput`

(prettier/prettier)